### PR TITLE
feat: adding character base class

### PR DESCRIPTION
**Pull request by kurtome**

### CODE REVIEW

File: examples/standard_platformer/lib/main.dart [hunk 1]
```diff
@@ -93,7 +93,7 @@ class ExamplePlatformerLeapGame extends LeapGame
     );
 
     player = Player();
-    world.add(player = Player());
```
Comment (erickzanardo) [inline examples/standard_platformer/lib/main.dart:96]: Nit: I would keep this line, and instead, remove the one above

**Review by erickzanardo — APPROVED** [review]
one minor thing, but already looking great!

File: examples/standard_platformer/lib/main.dart (hunk 1 above)
Comment (kurtome) [reply examples/standard_platformer/lib/main.dart:96]: Hm, but why? 😀

Personally not a fan of the assignment side effects

Comment (erickzanardo) [reply examples/standard_platformer/lib/main.dart:96]: Just cause one line less and we avoid the ! 

But this is just a minor bit, totally not blocking for the pr :)

**Pull request merged**